packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/advanced-duration.jsx

```diff
@@ -56,6 +58,9 @@ const AdvancedDuration = observer(
 
         const has_error = !!validation_errors?.duration?.length;
 
+        const duration_unit_text =
+            getUnitMap()[advanced_duration_unit].name_plural || getUnitMap()[advanced_duration_unit].name;
```

Comment:
@kate-deriv  let's introduce a variable for getUnitMap()[advanced_duration_unit] since it's used twice🙏🏻